openapi.yaml

```diff
@@ -235,59 +291,162 @@ components:
           items:
             type: number
             format: double
-        metadata:
-          type: object
-          properties:
-            datasource_names:
-              type: array
-              items:
+          description: >-
```

Comment:
Note: I deleted metadata here because I couldn't find it in any responses/docs from either valhalla or mapbox.